### DISCUSSION TITLE
Unsupported Mode

### DISCUSSION THREAD

**Discussion by Hideki-Okamoto-NA** [HumanSupportRobot/QandA#20] [General]
I am trying to start using HSRB which is remained by predecessor.
Once I pressed "start" button, then "Unsupported mode" message is displayed on the monitor of HSR.
Currently I don't know what is happening on HSR so that I am trying to execute clean install.
I wanna check the current version and also keep the backup just in case,
But I can not access by "ssh" (no answer for ping as well).

I would like to ask about how to access to HSR to confirm the version or taking the backup under the situation of above.

**Comment by hsr-admin** [comment]
First, connect an external monitor to the VGA port on the access panel by referring to the following
https://docs.hsr.io/hsrb_user_manual/overview/parts_names.html#access-panel

If the OS is running and commands can be entered on the monitor, you may be able to connect a USB stick and back up to it.

**Reply by Hideki-Okamoto-NA** [reply]
Thank you very much for your prompt response but even though I connected external monitor it is not solved yet.
Let me share current situation more detail.

When turn on the power switch then "Unsupported Mode " is shown on the display.
After a few second, some messages are displayed on black screen.
*The message is "[15.019944] usb 3-1.2: 3:1: cannot get min/max values for control 2 (id 3)"

Since not transition from the above status,  I press "Ctrl+Alt+Del" then the screen of "HSR Human Support Robot" is displayed.
Then start the OS process, I can move to MEBEX setup menu with "Ctrl+P" or entering setup with "Del" or BBS popup menu with "F7".
Without any key input start the loading Linux 4.4.47-rt59-tmc-2 and initializing the ramdisk.
And after "/dev/sda1: clean ~~~ blocks" displayed, then stop the process with black screen blinking cursor on top of left.

This is the current situation and even if connect the external monitor I can not enter on the monitor as of now.
